interface-definitions/service-mdns-repeater.xml.in

```diff
@@ -15,6 +15,31 @@
             <children>
               #include <include/generic-disable-node.xml.i>
               #include <include/generic-interface-multi.xml.i>
+              <leafNode name="ip-version">
+                <properties>
+                  <help>IP address version to use</help>
+                  <valueHelp>
+                    <format>_ipv4</format>
+                    <description>Use only IPv4 address</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>_ipv6</format>
+                    <description>Use only IPv6 address</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>both</format>
+                    <description>Use both IPv4 and IPv6 address</description>
+                  </valueHelp>
+                  <completionHelp>
+                    <list>ipv4 ipv6 both</list>
+                  </completionHelp>
+                  <constraint>
+                    <regex>(ipv[46]|both)</regex>
+                  </constraint>
+                  <constraintErrorMessage>IP Version must be literal 'ipv4', 'ipv6' or 'both'</constraintErrorMessage>
+                </properties>
+                <defaultValue>ipv4</defaultValue>
```

Comment:
To keep compatibility with previous versions this should be `both`

Reply:
Turns out, while the config file had `use-ipv6=yes`, the actual `conf_mode` operation [currently fails](https://github.com/vyos/vyos-1x/blob/6ffb104ada0af428ee293dc094a8f624725e9bbc/src/conf_mode/service_mdns-repeater.py#L63-L65) if the participating interfaces don't have IPv4 addresses.

So I am thinking `ipv4` might be a safer backward compatible default (for setups that currently do not have any IPv6 address(es) in the participating interfaces either because IPv6 is globally disabled or because `link-local` is disabled).

Reply:
Maintainer meeting came to the conclusion `both` is the real option here which must be used.

Reply:
Sure, changed the default to `both`.